dist/dattobd.spec

```diff
@@ -489,6 +509,11 @@ rm -rf %{buildroot}
 
 # Install systemd shutdown script
 %{_systemd_shutdown}/umount_rootfs.shutdown
+%{_systemd_services}/umount-rootfs.service
+
+
+%post 
+ln -s %{_systemd_services}/umount-rootfs.service   %{_systemd_services}/reboot.target.wants/umount-rootfs.service 
```

Comment:
what will happen if we delete dattobd installed this way? Will symbol links be deleted too?

Reply:
No, the original file and the link remain in the filesystem.

Reply:
I guess we should do something about it, we already have issues cause by not completly cleaning after dattobd when it is deleted